### DISCUSSION TITLE
[A-L] (2023/24) Foglio 4 - Esercizio 10

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#298] [Esercizi - Viaggi/Piazza]
![Screenshot 2023-10-28 110545](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/20b6492c-7df8-474b-802d-08aabf3bbb17)

**Comment by CiottoloMaggico** [comment]
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/79638284/7ee74f52-c664-4ea8-b574-f7e97ce0dda9)
